packages/fluent/scss/list/_theme.scss

```diff
@@ -0,0 +1,94 @@
+@use "../core/mixins" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-list--theme() {
+
+    .k-list {
+        @include fill(
+            var(--kendo-list-text, #{$kendo-list-text}),
+            var(--kendo-list-bg, #{$kendo-list-bg}),
+            var(--kendo-list-border, #{$kendo-list-border})
+        );
+    }
+
+
+    // List header
+    .k-list-group-sticky-header {
+        @include fill(
+            var(--kendo-list-header-text, #{$kendo-list-header-text}),
+            var(--kendo-list-header-bg, #{$kendo-list-header-bg}),
+            var(--kendo-list-header-border, #{$kendo-list-header-border})
+        );
+        @include box-shadow( var(--kendo-list-header-shadow, #{$kendo-list-header-shadow}) );
```

Comment:
I don't see a box-shadow in the designs, but I might be mistaken

Reply:
I discussed this with @imasyuko and we agreed that there should be a box-shadow on the sticky header element.